RELEASE_NOTES.md

```
@@ -8,6 +8,10 @@
 
     * [CP-3366] - Translation in Organisation related UI
 
+## Version 1.3.9 - Community 1.0.0
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)